kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -166,6 +170,131 @@ class DeltaTableWritesSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBa
     }
   }
 
+  test("create table and set properties") {
+    withTempDirAndEngine { (tablePath, engine) =>
+      val table = Table.forPath(engine, tablePath)
+      val txn1 = createTxn(engine, tablePath, isNewTable = true, testSchema, Seq.empty)
+
+      txn1.commit(engine, emptyIterable())
+
+      val ver0Snapshot = table.getSnapshotAsOfVersion(engine, 0).asInstanceOf[SnapshotImpl]
+      assertMetadataProp(ver0Snapshot, TableConfig.CHECKPOINT_INTERVAL, 10)
```

Comment:
This can be one utility method. Also repeated multiple times in tests

```
def setTablePropAndVerify(tablePath: String, isNewTable:Boolean = true, propKey: String, propValue: String) = {
    val table = Table.forPath(engine, tablePath)
      val txn1 = createTxn(engine, tablePath, isNewTable = true, testSchema, Seq.empty)

      txn1.commit(engine, emptyIterable())

      val ver0Snapshot = table.getSnapshotAsOfVersion(engine, 0).asInstanceOf[SnapshotImpl]
      assertMetadataProp(ver0Snapshot, TableConfig.CHECKPOINT_INTERVAL, 10)
}
```